### PR TITLE
New exercise `markdown`

### PR DESCRIPTION
**Pull request by meatball133**

### CODE REVIEW

**Comment by meatball133** [comment]
Realised this was a refactor exercise so will have to make this solution a lot uglier, will work on that and then put it as non draft

File: test.cr
Comment (ryanplusplus) [inline test.cr:1]: What is this file for?

File: exercises/practice/markdown/.meta/src/example.cr [hunk 1]
```diff
@@ -0,0 +1,29 @@
+class Markdown
+  def initialize(@markdown : String); end
+
+  def to_html
+    result = ""
+    lines = @markdown.split("\n")
+    lines.each_with_index do |line, index|
+      level = line.scan(/^#+/)
+      # TODO: Use present? method when Crystal1.10 is dropped
```
[the remark this answers is not in the document]
Comment (meatball133) [reply exercises/practice/markdown/.meta/src/example.cr:9]: Yeah, but is the ci running Crystal 1.8? I think

[the remark this answers is not in the document]
Comment (ryanplusplus) [reply exercises/practice/markdown/.meta/src/example.cr:9]: Yeah you're absolutely right. I was only thinking about the website workflow and not CI.

**Review by ErikSchierboom — APPROVED** [review]

**Pull request merged**